apps/issuer/models.py

```diff
@@ -583,7 +583,7 @@ def notify_issuer_owner(self, badgr_app=None, renotify=False):
                 'issuer_name': re.sub(r'[^\w\s]+', '', self.name, 0, re.I),
                 'issuer_url': self.url,
                 'issuer_email': self.email,
-                'site_name': badgr_app.name,
+                'site_name': re.sub(r'@', '', badgr_app.name),
```

Comment:
This is not very nice. We should either correct the email function or disallow an `@` sign altogether, I think. If you disagree though, you can merge I guess.

Reply:
Makes sense, in another PR i am disallowing problematic characters for the issuer name in the frontend now. However, we currently have an issuer on prod that has an "@" symbol in its name, which we cannot verify for this reason. My idea was to merge this change and revert it once this particular issuer is verified. It's not a good solution but i could not think of a better one, do you have another suggestion?